templates/CRM/common/civicrm.settings.php.template

```diff
@@ -647,7 +643,7 @@ if (!defined('CIVICRM_CLEANURL')) {
 // make sure the memory_limit is at least 64 MB
 $memLimitString = trim(ini_get('memory_limit'));
 $memLimitUnit   = strtolower(substr($memLimitString, -1));
-$memLimit       = (int) $memLimitString;
+$memLimit       = (int)$memLimitString;
```

Comment:
I think this one was originally correct as I think generally we want a space between the casting and the variable

Reply:
OK, I've fixed it.